client.go

```diff
@@ -93,6 +93,84 @@ func (c *Client) DiscoverVersions(versions []ProtocolVersion) (serverVersions []
 	return
 }
 
+// Create with the server
+func (c *Client) Create(objectType Enum, templateAttribute TemplateAttribute) (uniqueIdentifier string, err error) {
+	var resp interface{}
+	resp, err = c.Send(OPERATION_CREATE,
+		CreateRequest{
+			ObjectType:        objectType,
+			TemplateAttribute: templateAttribute,
+		})
+	if err != nil {
+		return "", err
+	}
+
+	uniqueIdentifier = resp.(CreateResponse).UniqueIdentifier
+	return
+}
```

Comment:
I think we are copying some bad style here. I'd suggest doing the following throughout these changes:
* Just take the `Create` struct as input
* Just return the `Response` struct as output
* Don't used named return values
* Check the result of casts
```suggestion
func (c *Client) Create(request CreateRequest) (CreateResponse, error) {
	resp, err := c.Send(OPERATION_CREATE, request)
	if err != nil {
		return CreateResponse{}, err
	}

	createResp, ok := resp.(CreateResponse)
	if !ok {
		return CreateResponse{}, errors.New("unexpected response type")
	}

	return createResp, nil
}
```